plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties

```diff
@@ -1,5 +1,5 @@
 extension.name=App Engine JRE Whitelist
 appengine.web=App Engine Web XML Validator
-blacklist.marker=Blacklist Marker
+appengine.blacklist.marker=App Engine Element Problem
```

Comment:
I think something that refers to the xml would be better, maybe `App Engine Configuration Problem`?